test/examples/content_filter.compose.yml

```diff
@@ -0,0 +1,16 @@
+# FASTDDS_TODO_BEFORE(3, 0, "This compose file should be used for the future configuration example");
```

Comment:
NIT
Pls remove this TODO and add the licence here and in the test.py file

```
# Copyright 2024 Proyectos y Sistemas de Mantenimiento SL (eProsima).
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.```

Reply:
I will apply your suggestion once the CI passed, so we don't need to re run it since this change is related to a commented part